features/attached_enable.feature

```diff
@@ -114,6 +114,32 @@ Feature: Enable command behaviour when attached to an UA subscription
            | trusty  | libgit2-0 | https://esm.ubuntu.com/ubuntu/      |
            | xenial  | libkrad0  | https://esm.ubuntu.com/infra/ubuntu |
 
+    @series.xenial
+    @series.bionic
+    @series.focal
+    Scenario Outline: Attached enable of a know service shows update in a ubuntu machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `ua enable esm-infra` with sudo
+        Then I will see the following on stdout:
+            """
+            One moment, checking your subscription first
+            ESM Infra is already enabled.
+            See: sudo ua status
+            """
+        When I run `apt install -y <pkg-version>` with sudo
+        And I run `apt update` with sudo
+        Then stdout matches regexp
+        """
+        \d* of the updates \w* from UA Infrastructure ESM
+        """
```

Comment:
this \d is going to be 0 for all but trusty right? Does this test do much then? Should it be testing the number on those just to note if we get a different answer we should be looking into it?

Reply:
Originally, I though that was the case. On focal that's exactly what is happening, but for bionic, just by enable `esm-infra`, we have 6 packages that appear in the esm update output. That's why I have used the regex. But I did not check if xenial would follow the same pattern

Reply:
ah ok, interesting. Can you let me know what those were? I'm curious if I'm mistaken that there's no value in esm-infra for bionic/xenial/focal after all then.

Reply:
No problem, I will reproduce the scenario on Bionic and Xenial and post the packages here

Reply:
Bionic:
```
krb5-locales/bionic-infra-security,bionic-infra-updates 1.16-2ubuntu0.1+esm1 all [upgradable from: 1.16-2ubuntu0.1]
libgssapi-krb5-2/bionic-infra-security,bionic-infra-updates 1.16-2ubuntu0.1+esm1 amd64 [upgradable from: 1.16-2ubuntu0.1]
libk5crypto3/bionic-infra-security,bionic-infra-updates 1.16-2ubuntu0.1+esm1 amd64 [upgradable from: 1.16-2ubuntu0.1]
libkrb5-3/bionic-infra-security,bionic-infra-updates 1.16-2ubuntu0.1+esm1 amd64 [upgradable from: 1.16-2ubuntu0.1]
libkrb5support0/bionic-infra-security,bionic-infra-updates 1.16-2ubuntu0.1+esm1 amd64 [upgradable from: 1.16-2ubuntu0.1]
```

Xenial:
```
krb5-locales/xenial-infra-security,xenial-infra-updates 1.13.2+dfsg-5ubuntu2.1+esm1 all [upgradable from: 1.13.2+dfsg-5ubuntu2.1]
libgssapi-krb5-2/xenial-infra-security,xenial-infra-updates 1.13.2+dfsg-5ubuntu2.1+esm1 amd64 [upgradable from: 1.13.2+dfsg-5ubuntu2.1]
libk5crypto3/xenial-infra-security,xenial-infra-updates 1.13.2+dfsg-5ubuntu2.1+esm1 amd64 [upgradable from: 1.13.2+dfsg-5ubuntu2.1]
libkrb5-3/xenial-infra-security,xenial-infra-updates 1.13.2+dfsg-5ubuntu2.1+esm1 amd64 [upgradable from: 1.13.2+dfsg-5ubuntu2.1]
libkrb5support0/xenial-infra-security,xenial-infra-updates 1.13.2+dfsg-5ubuntu2.1+esm1 amd64 [upgradable from: 1.13.2+dfsg-5ubuntu2.1]
```

Reply:
interesting ok. It looks like it's the same version that's in main already for those just without the +esm on the version string. Maybe the security team was testing something. They're in the list though so I guess it's not 0.